test/server.js

```diff
@@ -36,6 +36,6 @@ var server = function(req, res) {
 https.createServer({
   key: fs.readFileSync(path.join(__dirname, 'test-cdn.betterttv.net.key')),
   cert: fs.readFileSync(path.join(__dirname, 'test-cdn.betterttv.net.cert'))
-}, server).listen(443);
+}, server).listen(443, '127.0.0.1');
```

Comment:
Please remove the changes to this file from your PR. If you want to force localhost, please open a separate PR and justify why we should make this change. Some people may want to listen to all interfaces and this change is not related to your PR.